packages/contracts/test/framework/plugin/plugin-repo.ts

```diff
@@ -152,6 +153,23 @@ describe('PluginRepo', function () {
         expect(toProtocolVersion).to.deep.equal(osxContractsVersion());
       });
     });
+    describe('InitializeFrom', () => {
+      it('increments `_initialized` to `1`', async () => {
+        // Expect the contract to be initialized  with `_initialized = 1`.
+        expect(
+          ethers.BigNumber.from(
+            await ethers.provider.getStorageAt(
+              pluginRepo.address,
+              OZ_INITIALIZED_SLOT_POSITION
+            )
+          ).toNumber()
+        ).to.equal(1);
```

Comment:
This test now does two things at once. 
1.) it checks against an initialized contract (that was initialized by calling `initialize`, not `initializedFrom`) by checking `OZ_INITIALIZED_SLOT_POSITION`.
2) it checks that a function call to `initializeFrom` with version `[1,3,0]` as an argument fails.

Does 1) make sense if we `describe('initializeFrom',` ? I would argue: no. This should happen in the `describe('initialize',` test section.
Also `it('increments `_initialized` to `1`', async () => {` doesn't make clear that the expected behavior is to revert.

The osx repo unfortunately isn't a good place to look for orientation on how clean tests are written. Maybe take a look [here](https://github.com/aragon/osx-commons/blob/main/contracts/test/plugin/plugin-uups-upgradeable.ts#L31-L84) or in the different plugin repos.

Reply:
Implemented in c104eab